src/components/useFetch.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react'
+
+const useFetch = (url: string) => {
+    type post = {
+        userId: number;
+        id: number;
+        title: string;
+        body: string;
+    }
+    const [data, setData] = useState<Array<post>>([])
+    const [loading, setLoading] = useState<boolean>(true)
```

Comment:
The type is inherited from default value.
```suggestion
    const [loading, setLoading] = useState(true)
```

Reply:
Fixed.